utils_test.go

```diff
@@ -11,13 +11,13 @@ import (
 	"github.com/consensys/gnark/test"
 )
 
-type idTestCircuit struct {
+type IDTestCircuit struct {
```

Comment:
don't do this kind of change: just letter case. And in this case, we should use lower letter "id" as we don't need them in other files.